services/booking-api/test/lambdas/create.test.ts

```diff
@@ -109,6 +110,75 @@ it('creates a booking successfully', async () => {
   expect(result).toEqual(expectedResult);
 });
 
+it('includes a meeting link when remoteMeeting is true', async () => {
+  expect.assertions(1);
+
+  const searchResponseData: BookingSearchResponse['data'] = {
+    data: {
+      attributes: [
+        {
+          BookingId: '123456789',
+          Attendees: [
+            {
+              Type: 'Required',
+              Status: 'Declined',
+            },
+          ],
+        },
+      ],
+    },
+  };
+  const responseData = {
+    jsonapi: { version: '1.0' },
+    data: {
+      attributes: {
+        BookingId: '123456789',
+      },
+    },
+  };
+  const remoteMeetingResponseData = {
+    jsonapi: { version: '1.0' },
+    data: {
+      attributes: {
+        Id: 'string',
+        Subject: 'string',
+        OnlineMeetingUrl: 'remote.meeting.link',
+      },
+    },
+  };
+
+  getTimeSpans.mockResolvedValueOnce(getTimeSpansResponse);
+  mockedAreAllAttendeesAvailable.mockReturnValueOnce(true);
+  search.mockResolvedValueOnce({ data: searchResponseData });
+  mockedIsTimeSlotTaken.mockReturnValueOnce(false);
+  createRemoteMeeting.mockResolvedValueOnce({ data: remoteMeetingResponseData });
+  create.mockResolvedValueOnce({ data: responseData });
+
+  await main({ body: JSON.stringify({ ...mockBody, remoteMeeting: true }) }, mockContext);
+
+  expect(booking.create).toHaveBeenCalledWith(
+    expect.objectContaining({
+      body: expect.stringContaining(`htmltext
+      <div style="color:#252424; font-family:'Segoe UI','Helvetica Neue',Helvetica,Arial,sans-serif">
+        <div style="margin-top:24px; margin-bottom:20px">
+          <span style="font-size:24px; color:#252424">Microsoft Teams-möte</span>
+        </div>
+        <div style="margin-bottom:20px">
+          <div style="margin-top:0px; margin-bottom:0px; font-weight:bold">
+            <span style="font-size:14px; color:#252424">Jobba på datorn eller mobilappen</span>
+          </div>
+          <a href="remote.meeting.link"
```

Comment:
perhaps it's good enough with `href="remote.meeting.link"`

Reply:
Fixed